bindings/pyroot/pythonizations/python/ROOT/_pythonization/_tgraph.py

```diff
@@ -40,3 +43,63 @@ def set_size(self, buf):
 
 # Add the composite to the list of pythonizors
 cppyy.py.add_pythonization(comp)
+
+
+def _numpy_array(buffer, n):
+    # Helper to create a numpy array from a raw array pointer.
+    #
+    # Note: The data is copied.
+    #
+    # Args:
+    #     buffer (cppyy.LowLevelView):
+    #         The pointer to the beginning of the array data, usually
+    #         obtained from a C++ function that returns a `double *`.
+    #
+    # Returns:
+    #     numpy.ndarray
+    import numpy as np
+
+    if not buffer:
+        return None
+
+    a = np.copy(np.frombuffer(buffer, dtype=np.float64, count=n))
+    return a
+
+
+def _TGraphConstructor(self, *args):
+    import numpy as np
+
+    if isinstance(args[0], (np.ndarray, np.generic)):
+        args = (args[0].size,) + args
+
+    self._original__init__(*args)
+
+
+def Get(self):
+    a = _numpy_array(self._Original_GetX(), self.GetN())
+    b = _numpy_array(self._Original_GetY(), self.GetN())
+    return a, b
```

Comment:
```suggestion
def Get(self):
    return self.GetX(), self.GetY()
```

To avoid code repetition?